docs/cpp/cmake-linux.md

```diff
@@ -0,0 +1,176 @@
+
+The CMake Tools extension integrates Visual Studio Code and CMake to make it easy to configure, build, and debug your C++ project using CMake.
+
+In this tutorial, you'll use the CMake Tools extension for Visual Studio Code to configure, build, and debug a simple C++ CMake project on Linux. Aside from installing CMake and your compiler, debugger, the steps in this tutorial apply generally to how you'd use CMake on other platforms, like Windows.
+
+If you have any trouble, please file an issue for this tutorial in the [VS Code documentation repository](https://github.com/Microsoft/vscode-docs/issues).
+
+## Prerequisites
+
+To successfully complete this tutorial on Ubuntu, install the following:
+
+1. [Visual Studio Code](/download).
+1. [C++ extension for VS Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools). You can install the C/C++ extension by searching for 'c++' in the Extensions view (`kb(workbench.view.extensions)`).
+
+    ![C/C++ extension](images/cpp/cpp-extension.png)
+
+1. [CMake Tools extension for VS Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cmake-tools). You can install the C/C++ extension by searching for 'cmake tools' in the Extensions view (`kb(workbench.view.extensions)`).
+
+    ![CMake tools extension](images/cpp/cmake-extension.png)
+
+1. You'll also need to install CMake, a compiler, a debugger, and a generator.
+
+### Ensure that CMake is installed
+
+The VS Code CMake Tools extension does its work using CMake installed on your system. For best results, use CMake version 3.17 or greater.
+
+See if CMake is already installed on your system. Open a Terminal window and enter the following command:
+
+```bash
+cmake --version
+```
+
+To install, or get the latest version of CMake on your system, see the instructions for your platform on the [Kitware APT Repository](https://apt.kitware.com/). You want to install a version greater than 3.17.
+
+### Ensure that development tools are installed
+
+Although you'll use VS Code to edit your source code, you'll compile and debug the source code using the compiler, debugger, and build tools (such as make) installed on your system.
+
+For this tutorial on Ubuntu, we'll use the GCC compiler, GDB to debug, and make to genenerate native build files. These tools are not installed by default on Ubuntu, so you need to install them. Fortunately, that's easy.
+
+### Check if GCC is installed
+
+ To see if GCC is already installed on your system, open a Terminal window and enter the following command:
+
+```bash
+gcc -v
+```
+
+If GCC isn't installed, run the following command from the terminal window to update the Ubuntu package lists. An out-of-date Linux distribution can interfere with getting the latest packages.
+
+```bash
+sudo apt-get update
+```
+
+Next, install the GNU compiler, make, and the GDB debugger with this command:
+
+```bash
+sudo apt-get install build-essential gdb
+```
+
+## Create a CMake quickstart
```

Comment:
This reads awkwardly to me.  I expected it to read like "Quick start: Create a CMake project" or just "Create a CMake project".

Reply:
Not my wording. Changed to 'Create a CMake project'